src/extras/Post.js

```diff
@@ -106,7 +106,7 @@ export class Post {
             this.gl.renderer.render({
                 scene: pass.mesh, 
                 target: i === enabledPasses.length - 1 && (target || !this.targetOnly) ? target : this.fbo.write,
-                clear: i === enabledPasses.length - 1 ? true : false,
+                clear: i === true,
```

Comment:
Can either remove this line as it auto clears by default, or remove the `i ===`

Reply:
Oh yes, I made a mistake. 😝Fixed.